README.md

```diff
@@ -1,110 +1,169 @@
 
-![illuminations-1.0-fireflies](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.0-fireflies.png)
+![Fireflies](https://user-images.githubusercontent.com/83953120/123544977-2ef68800-d756-11eb-9356-6394353ce16f.png)
 
-### Chorus Petals
-
-Appearing near Chorus flowers, these beautiful petals will twirl down to the ground, to the winds of the End, transitioning from a beautiful white to a majestic purple. The younger the flower is, the more petals will appear, and breaking a Chorus flower will provoke a burst of petals (again, the younger the flower, the more petals will burst).
-
-![illuminations-1.4-choruspetals](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.4-choruspetals.png)
-
-### Glowworms
+#### Glowworms
 
 **Glowworms** are small larvae living in caves by sticking to the ceiling that will fall down if their support block is removed. They will appear in small quantities in jungle, plains, savanna and taiga type biomes, medium quantities in forest and river type biomes and great quantity in swamp type biomes.
 
-![illuminations-1.0-glowworms](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.0-glowworms.png)
+![Glowworms](https://user-images.githubusercontent.com/83953120/123544983-328a0f00-d756-11eb-8e51-2ee132c07cc2.png)
 
-### Plankton
+#### Plankton
 
 Glowing **plankton** - even though difficult to notice because of its size - can be found floating around in the dark parts of the oceanic depths. They appear in any ocean biome as long as the light is sufficiently low.
 
-![illuminations-1.0-plankton](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.0-plankton.png)
+![Plankton](https://user-images.githubusercontent.com/83953120/123544988-374ec300-d756-11eb-9c1a-8d427c9b3916.png)
 
-### "Autumnflies"
+### Halloween (October Specials)
+
+#### Autumnflies
 
 During the month of October, fireflies will adopt an orange tint instead of their regular green one.
 
-![illuminations-1.2-autumnflies](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.2-autumnflies.png)
+![Autumnflies](https://user-images.githubusercontent.com/83953120/123544993-3ddd3a80-d756-11eb-9389-1bc08f971f0b.png)
 
-### Eyes in the Dark
+#### Eyes in the Dark
 
 Also during the month of October, you may come across glowing eyes in absolutely dark spots. These eyes will disappear if you come too close or if their darkness is disturbed by a light source.
 
-![](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.2-eyes.png)
+![EyesInTheDark](https://user-images.githubusercontent.com/83953120/123545003-42095800-d756-11eb-84a3-7f2e43843acc.png)
```

Comment:
Screenshot could be better. We only see two eyes, yet the screenshot takes all this place. I suggest making a screenshot in an environment with light and darkness, and show the eyes appear in this darkness.